src/Sulu/Bundle/AdminBundle/Resources/js/components/Select/Select.js

```diff
@@ -128,7 +257,12 @@ class Select<T> extends React.Component<Props<T>> {
         const clonedChildren = this.cloneChildren();
 
         return (
-            <div className={selectStyles.select}>
+            <div
+                className={selectStyles.select}
+                onKeyDown={this.handleKeyDown}
```

Comment:
is there a specific reason for binding `onKeyDown` and `onKeyPress`? 🙂

Reply:
keyPress ignores non printable keys, and that's exactly what is needed ... Since arrow down/up and escape are not captured by keyPress, I need the keyDown as well

Reply:
ah okay did not know this 👍